roles/satellite-clone/tasks/main.yml

```diff
@@ -19,6 +19,20 @@
     keys: [":foreman_url"]
   register: backup_hostname
 
+- name: Get candlepin password
```

Comment:
I'm not sure how many times we do this but it seemed a bit slow (I hope I'm not confusing it w/ another step). Maybe we can just load the entirety of any yamls we need once, and refer to any keys later directly in memory?

Reply:
Do you have any numbers on how long it took? It was around 1 second for me but it was a really small backup, I am curious how it's performing on a large backup. I'm sure we could improve it but will have to see what it takes.